Classes/Issues/Comments/Reactions/IssueCommentReactionCell.swift

```diff
@@ -156,6 +156,23 @@ UICollectionViewDelegateFlowLayout {
         let model = reactions[indexPath.item]
         cell.label.text = "\(model.content.emoji) \(model.count)"
         cell.contentView.backgroundColor = model.viewerDidReact ? Styles.Colors.Blue.light.color : .clear
+        
+        var users = model.users
+        guard users.count > 0 else { return cell }
+        
+        let difference = model.count - users.count
+        if difference > 0 {
+            let plural = difference > 1 ? "s" : ""
+            users.append("\(difference) other\(plural)")
+        }
+        
+        let lastUser = users.removeLast()
+        var message = users.joined(separator: ", ")
+        message += users.count > 0 ? " and " + lastUser : lastUser
+        message += " reacted with " + model.content.localizedString + " emoji"
```

Comment:
Dropped the emoji name, will work on the localization formatting (never done it before!)